package.json

```diff
@@ -101,5 +103,10 @@
       "spy",
       "expect"
     ]
+  },
+  "nyc": {
```

Comment:
Code coverage tool I'm testing - istanbul.
I've also tested on one of my open-source projects integration of CircleCI with CodeCov which is tool for presenting the coverage information:
https://codecov.io/gh/michaloo